source/assets/js/components/sass-syntax-switcher.ts

```diff
@@ -36,14 +36,18 @@ $(() => {
       if (!noRecursion) {
         noRecursion = true;
         jqASass.not(this).trigger('click');
+        $('body').attr('data-selected-syntax', 'sass');
         noRecursion = false;
       }
     }),
     jqASCSS = jqA.filter(":contains('SCSS')").on('click', function () {
       if (!noRecursion) {
         noRecursion = true;
         jqASCSS.not(this).trigger('click');
+        $('body').attr('data-selected-syntax', 'scss');
         noRecursion = false;
       }
     });
+    // Set initial value
+    $('body').attr('data-selected-syntax', 'scss');
```

Comment:
I initially had a `:has` selector to figure out which syntax was selected, but realized I could set it explicitly.